docs/apps/lnw/es2k/es2k-linux-networking.md

```diff
@@ -280,7 +277,7 @@ If the packets coming from a remote machine to the physical port are not VxLAN t
 ## Summary
 
 - Verification of source port and Associated L2 Bridge: The P4 Control Plane (P4 CP) must ensure the validation of the source port and its corresponding L2 bridge before initiating any further regulation of datapath packet classification.
-- Exception Packet Handling for All Protocols: The P4 Control Plane (P4 CP) shall incorporate exception packet handling logic, not limited to ARP but applicable to the first packet of any protocol.
+- Exception Packet Handling for all Protocols: The P4 Control Plane (P4 CP) shall incorporate exception packet handling logic, not limited to ARP but applicable to the first packet of any protocol.
```

Comment:
Do Packet, Handling and Protocols need to be capitalized?

Reply:
not sure, if this is how originally this document was written. For example on line 279 capital `A` is used for Associated. What have we followed recently in our docs?

Reply:
I have updated it to not have capitalized words in the middle of a sentence

Reply:
Reading the doc on the p4cp-userguide, it does seem odd that those are capitalized. Capitalizing first letter of words is ok for headings, but not during normal sentence construction. We should fix it to all lowercase in the text (non-headings)